src/routes/routesConfig.tsx

```diff
@@ -0,0 +1,29 @@
+import { createBrowserRouter } from "react-router-dom";
+import MainLayout from "../layouts/MainLayout";
+import HomePage from "../pages/Home/Home";
+import LoginPage from "../pages/Login/Login";
+import RegistrationPage from "../pages/Registration/Registration";
+import NotFoundPage from "../pages/NotFound/NotFound";
+
+const router = createBrowserRouter(
+  [
+    {
+      path: "/",
+      Component: MainLayout,
+      children: [
+        { index: true, Component: HomePage },
+        { path: "login", Component: LoginPage },
+        { path: "register", Component: RegistrationPage },
```

Comment:
you've created more sample pages than listed here, so I'd recommend to adjust the pages here as well. See https://reactrouter.com/start/declarative/routing for the info of applying different layouts, use dynamic routing for product pages, etc